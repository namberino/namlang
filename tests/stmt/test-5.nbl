// no initializers
var a;
var b;

a = "assigned";

print(a); // works
print(b); // error
