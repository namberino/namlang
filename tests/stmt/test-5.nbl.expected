Executing file: tests/stmt/test-5.nbl
assigned
nil
